.github/filters.yml

```
@@ -10,17 +10,12 @@ ingestion_server:
   - ingestion_server/**
   # Change to the CI + CD workflow should trigger complete workflow.
   - .github/workflows/ci_cd.yml
-frontend:
-  - frontend/**
-  - package.json
-  - pnpm-lock.yaml
-  - pnpm-workspace.yaml
-  - .npmrc
-  - .pnpmfile.cjs
-  # Change to the CI + CD workflow should trigger complete workflow.
-  - .github/workflows/ci_cd.yml
 documentation:
   - documentation/**
+  - frontend/**.mdx
+  - frontend/.storybook
```

I don't think `filters.yml` works with directories.
```suggestion
  - frontend/.storybook/**
```